jenkins-tools/yast-jenkins-modify.rb

```diff
@@ -17,17 +17,37 @@
 URL_BASE = "https://#{USER}:#{PWD}@ci.opensuse.org"
 #URL_BASE = "http://river.suse.de"
 
+# modules that do not follow yast-{mod} convention
+SPECIAL_MOD_NAMES = [
+  "skelcd-control-SLES",
+  "skelcd-control-SLED",
+  "skelcd-control-openSUSE",
+  "skelcd-control-openSUSE-LangAddOn",
+  "skelcd-control-SLES-for-VMware",
+  "y2r"
+]
```

Comment:
ugh? in current way no. As jenkins jobs have different format then github one, as on jenkins we share namespece with rest of jobs, so we need yast prefix there.
Hardcoded list is currently the easiest and quicker solution for me.